src/components/all.sass

```diff
@@ -48,5 +49,12 @@ $body-color: #333
   //padding-left: 20px
   color: #000000
 
+.horizontal-rule
+  background-color: $tech-connect-blue
+  height: 4px
+
+.blue-text
```

Comment:
I'm not a fan of this name - what if "someone in Branding" decides that it should be grey? I _can_ just change the value of the color, but the code will be extremely confusing to maintain. Additionally, we're just setting it to the primary color value, so why not just use Bulma built-ins? I believe `has-text-primary` would have the exact same effect with more portability.

- [ ] remove `blue-text` from sass
- [ ] change `blue-text` to `has-text-primary` in all js files